lib/js/atoms/Stepper.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<div class="stepper">
+		<div v-for="(step, index) in steps" :key="index" class="stepper__item">
+			<div
+				v-if="index !== 0"
+				class="stepper__separator"
+				:class="{ '-active': step.isActive }"
+			></div>
+			<div class="stepper__icon" :class="{ '-active': step.isActive }">
+				<icon :icon="ICONS[step.iconKey]" :size="ICON_SIZES.X_SMALL"></icon>
+			</div>
+			<div class="stepper__label" :class="{ '-active': step.isActive }">
+				{{ step.label }}
+			</div>
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../styles/settings/spacings';
+@import '../../styles/settings/colors';
+@import '../../styles/settings/media-queries';
+@import '../../styles/settings/typography';
+
+.stepper {
+	display: flex;
+
+	&__item {
+		align-items: center;
+		display: flex;
+		justify-content: center;
+	}
+
+	&__label {
+		@include textInfoM();
+		@include textBold();
+
+		color: $color-minor;
+
+		&.-active {
+			color: $color-primary;
+		}
+	}
+
+	&__icon {
```

Comment:
do we need additional wrapper on icon?
Maybe we could apply styles on `<wnl-icon class="stepper__icon">`? This way you could avoid setting div size